docs/sources/reference/playlist.md

```diff
@@ -11,59 +11,95 @@ weight = 4
 
 ## Start a playlist
 
-Also, if you want, you can start the playlist without saving it by clicking on the green **Start** button at the bottom.
+To start a playlist: 
 
-## Control a playlist
+1. From the Dashboard submenu, click __Playlists__.
+2. Next to the playlist you want to start, click __Start Playlist__.
+3. In the dropdown, select the mode you want the playlist to display in.
+   - In Normal mode
+   - In TV mode
+   - In TV mode (with auto fit panels)
+   - In Kiosk mode
+   - In Kiosk mode (with auto fit panels)
+
+### Normal, TV and Kiosk mode
+
+__Normal mode:__ The side menu remains visible. The navbar, row and panel controls appear at the top of the screen.
+
+__TV mode:__ The side mmenu is hidden/removed. The navbar, row and panel controls appear at the top of the screen.
 
-Playlists can also be manually controlled utilizing the Playlist controls at the top of screen when in Playlist mode.
+A playlist enters __TV mode__ automatically after one minute of user inactivity. To toggle it manually, 
+use the `d v` sequence shortcut, or append the parameter `?inactive` to the dashboard URL. Restore the navbar and controls with any mouse movement or keyboard action.
 
-Click the stop button to stop the Playlist, and exit to the current Dashboard.
-Click the next button to advance to the next Dashboard in the Playlist.
-Click the back button to rewind to the previous Dashboard in the Playlist.
+__Kiosk mode:__ The side menu, navbar, row and panel controls are completely hidden/removed from view. 
+
+To put a playlist into __Kiosk__ mode, use the `d k` shortcut after the playlist has started. Toggle the playlist out of kiosk mode with the same shortcut.
+
+## Control a playlist
 
-## TV or kiosk mode
+To control a playlist:
 
-In TV mode the top navbar, row and panel controls will all fade to transparent.
+1. While in __Normal__ or __TV__ mode, locate the Playlist navbar and controls at the top of the screen.
 
-This happens automatically after one minute of user inactivity but can also be toggled manually
-with the `d v` sequence shortcut, or by appending the parameter `?inactive` to the dashboard URL. Any mouse movement or keyboard action will
-restore navbar and controls.
+   Note: A playlist cannot be controlled manually in __Kiosk__ mode.
 
-Another feature is the kiosk mode - in kiosk mode the navbar is completely hidden/removed from view. This can be enabled with the `d k`
-shortcut.
+2. Click the __next__ button (right arrow) to advance to the next dashboard. 
+3. Click the __back__ button (left arrow) in the navbar to return to the previous dashboard. 
+4. Click the square __Stop__ button to stop the playlist. This will exit to the current Dashboard.
 
-To put a playlist into kiosk mode, use the `d k` shortcut after the playlist has started. The same shortcut will toggle the playlist out of kiosk mode.
+>Shortcut: Click the __Esc__ key to stop the playlist from your keyboard.
```

Comment:
According to the [Microsoft Style Guide](https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/term-collections/keys-keyboard-shortcuts), we shouldn't make keyboard shortcuts bold.